src/traces/sunburst/calc.js

```diff
@@ -192,7 +192,9 @@ exports.calc = function(gd, trace) {
     if(failed) return;
 
     // TODO add way to sort by height also?
-    hierarchy.sort(function(a, b) { return b.value - a.value; });
+    if (trace.sort) {
```

Comment:
Please remove space after if statement.
It should look like:
```js
if(trace.sort) {
```

You could also try 
```
npm run test-syntax
```
before committing.

Reply:
CirlcleCi mentioned effectively the additional space. I should have noticed that too.

But the test I ran locally is giving 100s of lines of .. has incorrect header information.., but nothing about the linting comments. I have to figure out if the local install is missing something.

Also for other tests I read I should have docker in place. All by all a steep learning curve ..